README.md

```diff
@@ -91,8 +91,8 @@ along existing investments, codebases, and developer populations. There are a
 few languages that have followed this model for other ecosystems, and Carbon
 aims to fill an analogous role for C++:
 
--   JavaScript → TypeScript
--   Java → Kotlin
+-   JavaScript → **_TypeScript_**
+-   Java → **_Kotlin_**
```

Comment:
I think the intent here is to put "Carbon" in boldface to emphasize that it's the part this project is adding. Making the other languages boldface would defeat the purpose.